ecosystem/indexer-grpc/transaction-filter/src/filters/move_module.rs

```diff
@@ -28,11 +29,31 @@ pub struct MoveStructTagFilter {
     pub module: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub name: Option<String>,
+    #[serde(skip)]
+    #[builder(setter(skip))]
+    standardized_address: OnceCell<Option<String>>,
```

Comment:
I wonder if it'd be cleaner to use a new `AccountAddressWrapper` type that contains the standardized version of the string. but on first construction since it seems you're doing it lazily for efficiency, maybe you don't standardize yet, you only do it on first access, and store a bool like `has_been_standardized`.

Reply:
Spoke offline, we'll keep as is so that we can retain the original address passed in if the address is ever serialized